packages/@aws-cdk/aws-ecr-assets/test/dockerignore-image/.dockerignore

```diff
@@ -1 +1,3 @@
 foobar.txt
+Dockerfile
```

Comment:
For reference these added lines are what cause the existing tests to also validate the correctness of this bug fix

Reply:
Yeah this great, but it didn't cover the custom docker file scenario. I saw you did add a `.dockerignore` for the custom test case as well, which is great 💯